src/components/NodeLink/functionUpdateVis.ts

```diff
@@ -371,28 +431,45 @@ export function drawNested(
 
   // Set some bar specific variables that we'll need for tracking position and sizes
   let i = 0;
-  const barWidth = glyphVariables.length === 0 ?
-    nodeMarkerLength / barVariables.length :
-    (nodeMarkerLength / 2) / barVariables.length;
+  const barWidth =
+    glyphVariables.length === 0
+      ? nodeMarkerLength / barVariables.length
+      : nodeMarkerLength / 2 / barVariables.length;
 
   for (const barVar of barVariables) {
-    const maxValue: number = parseFloat(max(graphStructure.nodes.map((n: Node) => parseFloat(n[barVar]))) || '');
+    const maxValue: number = parseFloat(
+      max(graphStructure.nodes.map((n: Node) => parseFloat(n[barVar]))) || '',
+    );
     // Draw white, background bar
-    node.append('rect')
+    node
+      .append('rect')
       .attr('class', 'bar')
       .attr('width', `${barWidth - 10}px`)
       .attr('height', `${nodeMarkerHeight - 16 - 5 - 5}px`)
-      .attr('y', `${16 +  5}px`)
-      .attr('x', `${5 + (i * barWidth)}px`)
+      .attr('y', `${16 + 5}px`)
+      .attr('x', `${5 + i * barWidth}px`)
       .style('fill', '#FFFFFF');
 
     // Draw the color bar with height based on data
-    node.append('rect')
+    node
+      .append('rect')
       .attr('class', 'bar')
       .attr('width', `${barWidth - 10}px`)
-      .attr('height', (d: Node) => `${(nodeMarkerHeight - 16 - 5 - 5) * d[barVar] / maxValue}px`)
-      .attr('y', (d: Node) => `${nodeMarkerHeight - 5 - ((nodeMarkerHeight - 16 - 5 - 5) * d[barVar] / maxValue)}px`)
-      .attr('x', `${5 + (i * barWidth)}px`)
+      .attr(
+        'height',
+        (d: Node) =>
+          `${((nodeMarkerHeight - 16 - 5 - 5) * d[barVar]) / maxValue}px`,
+      )
+      .attr(
+        'y',
+        (d: Node) =>
+          `${
+            nodeMarkerHeight -
+            5 -
+            ((nodeMarkerHeight - 16 - 5 - 5) * d[barVar]) / maxValue
+          }px`,
```

Comment:
When you see the formatter making changes such as this, it's a good clue that some sort of refactoring is in order.

In this case, I'd recommend defining a `const` variable to compute that expression with the `- 16 - 5 - 5` part so that the final value of the function can be expressed more concisely. That sequence of subtractions is so odd that I would include a comment explaining it. And finally, the two instances of that expression occur in functions that are almost identical--maybe there's something that can be done there as well (maybe not).

Just to be clear, this is an example of the sort of general advice I'd give upon seeing weird formatting like this. Don't make those recommended changes on this branch (since this branch is just meant for getting the prettier infrastructure into place).